cmake/CeleritasUtils.cmake

```diff
@@ -43,15 +43,180 @@ macro(celeritas_find_package_config _package)
 endmacro()
 
 function(celeritas_link_vecgeom_cuda target)
-  set_target_properties(${target} PROPERTIES
-    LINKER_LANGUAGE CUDA
-    CUDA_SEPARABLE_COMPILATION ON
-  )
+return()
+
+#  Readd when using
+  #set_target_properties(${target} PROPERTIES
+  #  LINKER_LANGUAGE CUDA
+  #  CUDA_SEPARABLE_COMPILATION ON
+  #)
+
+  # Note: the repeat (target name, location) below is due the author of  cuda_add_library_depend
+  # not knowing how to automatically go from the target to the real file from a generator expression in add_custom_command
```

Comment:
Can you also note that `cuda_add_library_depend` is actually a macro defined by vecgeom, not a standard CMake or FindCUDA macro? I had to google it (which incidentally cause google to recommend that I get some male diapers).

Reply:
This will go away.